<?xml version="1.0"?>
<launch>
  <!-- start environment modelling -->
  <!--node pkg="nodelet" type="nodelet" name="cam3d_nodelet_manager"  args="manager" output="screen"/-->


	<node pkg="cob_table_object_cluster" type="table_object_cluster_node" name="table_object_cluster" output="screen">
	  <remap from="/get_point_map" to="/point_map/get_map"/>
		<param name="file_path" type="string" value="/home/goa/tmp/"/>
  	<param name="save_to_file" type="bool" value="false"/>
  	<param name="height_min" type="double" value="-0.5"/>
    <param name="height_max" type="double" value="-0.03"/>
    <param name="min_cluster_size" type="int" value="10"/>
    <param name="cluster_tolerance" type="double" value="0.06"/>
  </node>


</launch>

